pages/index.tsx

```diff
@@ -24,13 +25,23 @@ const HomePage: React.FC<InferGetStaticPropsType<GetStaticProps>> = () => (
       <StackItem
         isFilled
         style={{
-          backgroundColor: 'var(--pf-global--BackgroundColor--100)',
+          backgroundColor: 'var(--pf-global--BackgroundColor--200)',
           color: 'var(--pf-global--Color--100)',
           padding: '3rem',
         }}
       >
         {WhyDevfiles}
       </StackItem>
+      <StackItem
+        isFilled
+        style={{
+          backgroundColor: 'var(--pf-global--BackgroundColor--100)',
+          color: 'var(--pf-global--Color--100)',
+          padding: '3rem',
+        }}
+      >
+        {ValueProps}
```

Comment:
May want to reduce the padding spaces between why devfiles and value prop. It is a little too big for the time being. Reduce by 20% will be fine.